config/identify_system.go

```diff
@@ -82,6 +82,15 @@ func identifySystem(config ServerConfig) (systemID string, systemType string, sy
 		if systemScopeFallback == "" {
 			systemScopeFallback = selfManagedSystemScope(config)
 		}
+	} else if (config.TemboInstanceID != "") || systemType == "tembo" {
+		systemType = "tembo"
+		// TODO(ianstanton) These should be tembo specific
```

Comment:
As discussed, something like this would make sense (in our call we considered whether it could make sense to auto-detect this from the hostname, but given that the hostname does not include these identifiers, and the collector is configured through Tembo, it seems better to require these to be set in the collector config).

```golang
		if systemID == "" {
			systemID = config.TemboInstanceID
		}
		if systemScope == "" {
			systemScope = config.TemboOrgID
		}
```

Reply:
Updated